src/locales/default_sv.json

```diff
@@ -55,6 +55,10 @@
       "undo": "Ångra",
       "undidMess": "Ångrad publicering"
     },
+    "weather": {
```

Comment:
Maybe use Temp or Temperatur instead of nu. Nu feels a bit weird :)

Reply:
took it away